frontend/packages/dev-console/src/components/import/app/ApplicationSelector.tsx

```diff
@@ -48,7 +48,6 @@ const ApplicationSelector: React.FC<ApplicationSelectorProps> = ({ namespace })
           label="Application"
           helperTextInvalid={errorMessage}
           isValid={isValid}
-          isRequired
```

Comment:
Missed this change in this PR https://github.com/openshift/console/pull/2550#issuecomment-526789480